src/app/file-manager/login/login.component.ts

```diff
@@ -43,6 +43,10 @@ export class LoginComponent implements OnInit {
     );
   }
 
+  openFcOnBrowser() {
+    this.loginService.openLink(this.fireCloudURL);
```

Comment:
Is there a reason why you didn't inject `ElectronService` directly into this component? It seems odd to add an API to `LoginService` that isn't a login related operation.

Reply:
Actually i was trying to avoid unnecessarily inject `ElectronService` into the component because i though the opposite would be odd. But from your point of view, you're right. I will change it as you recommend.